website/blog/2024-03-18-deploying-offline-with-docker/index.md

```diff
@@ -0,0 +1,69 @@
+---
+authors: [wwayne]
+tags: [deployment]
+---
+
+# Deploying offline with Docker
+
+<div align="center">
+
+![No internet access](./no-internet.png)
+
+</div>
+
+Are you facing a situation where your work environment lacks internet access, and you're wondering if you can still deploy Tabby? Fear not, because the answer is YES! 🐱📣
```

Comment:
```suggestion
Are you working in an air-gapped environment, and wondering if you can still deploy Tabby? Fear not, because the answer is YES! 🐱📣
```

Reply:
updated